packages/clerk-js/src/ui/components/Checkout/useCheckout.ts

```diff
@@ -24,7 +24,9 @@ export const useCheckout = (props: __internal_CheckoutProps) => {
       planPeriod,
       ...(subscriberType === 'org' ? { orgId: organization?.id } : {}),
     },
-    undefined,
+    {
+      staleTime: 0,
+    },
```

Comment:
This seems like an odd way to fix. Should we clear the checkout fetch cache on checkout success instead?

Reply:
We do invalidate on checkout but it's tied to the selected checkout that was completed, AFAI we don't have a way to clear the cache in the way our `useFetch` is working, @panteliselef will work on replacing our `useFetch` with [SWR](https://swr.vercel.app/) that will probably solve this issue without this addition

Reply:
👍 can we leave a comment here with additional context?

Reply:
Added!